session.go

```diff
@@ -65,8 +65,7 @@ type session struct {
 	reader Reader // @reader should be nil when @conn is a gettyWSConn object.
 	writer Writer
 
-	// read & write
-	rQ chan interface{}
+	// write
```

Comment:
Hey, guy, u can add "// DEPRECATED" on this func, pls do not delete it. U can refer to https://github.com/alexstocks/getty/blob/master/getty.go#L153.

Reply:
done